src/actions.ts

```diff
@@ -123,6 +136,45 @@ export const goToLineBoundary = (editor: Editor, boundary: 'start' | 'end') => {
   }
 };
 
+export const navigateLine = (editor: Editor, direction: 'up' | 'down') => {
+  const pos = editor.getCursor();
+  let line;
```

Comment:
_nit_: would prefer adding a type if it's not initialised, i.e. `let line: number`